eng/testing/linker/project.csproj.template

```diff
@@ -40,7 +47,6 @@
     <PackageReference Include="Microsoft.NET.ILLink.Tasks"
                       Version="{MicrosoftNETILLinkTasksVersion}"
                       PrivateAssets="all"
-                      IsImplicitlyDefined="true"
```

Comment:
Why is this removed?

Reply:
I removed it because it was throwing a warning for me before, but it looks like t hat was fixed by Viktor's recent change so I'll undo this change.

---
In reply to: [578724130](https://github.com/dotnet/runtime/pull/48429#discussion_r578724130) [](ancestors = 578724130)